settings.local.generic.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Local Backdrop CMS configuration file.
+ */
+
+/**
+ * Simple database settings:
+ *
+ * Most sites can configure their database by entering the connection string
+ * below. If using master/slave or multiple connections, see the advanced
+ * database settings.
+ */
+require_once __DIR__ . '/civicrm_table_prefixes.php';
+$databases = array(
+  'default' => array (
+    'default' =>
+    array (
+      'driver' => 'mysql',
+      'database' => 'backdropcmsorg',
+      'username' => '',
+      'password' => '',
+      'host' => '127.0.0.1',
+      'prefix' => $civicrm_table_prefixes,
+      'charset' => 'utf8mb4',
+      'collation' => 'utf8mb4_general_ci',
+    ),
+  ),
+);
+
+/**
+ * Site configuration files location.
+ */
+$config_directories['active'] = '../config/dev-active';
+$config_directories['staging'] = '../config/staging';
+
+/**
+ * Trusted host configuration (needed to avoid a status warning).
+ */
+$settings['trusted_host_patterns'] = array(
+  '', // this should be your local site, like '^backdropcms\.org$'
```

Comment:
Maybe we can set this to `$settings['trusted_host_patterns'] = array('.*');` to save everyone the headache of figuring out how to provide their local domain in the correct format?

Reply:
Nice trick, I'll use that myself. (Will make the change.)